src/components/LendCatalogue.tsx

```diff
@@ -97,7 +114,12 @@ const LendCatalogue: React.FC<LendCatalogueProps> = ({ data }) => {
                     className="Product__details"
                     style={{ marginTop: "8px" }}
                   >
-                    <LendButton id={face.id} handleLend={handleLend} />
+                    {!iLend ? (
```

Comment:
just one thing stop lending tx will pass only if the nft is not currently rented by anyone just a check if we wanna add on front otherwise would be handled on contract side